pkg/web_app/lib/src/admin_pages.dart

```diff
@@ -0,0 +1,477 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:html';
+
+import 'package:_pub_shared/data/account_api.dart';
+import 'package:_pub_shared/data/package_api.dart';
+import 'package:_pub_shared/data/publisher_api.dart';
+
+import '_authentication_proxy.dart';
+import '_dom_helper.dart';
+import 'api_client/api_client.dart' deferred as api_client;
```

Comment:
I think that using the same `api_client.dart` as deferred will only create a single split point, regardless of how many places use it. We should rather concentrate the heavy libraries after a split point barrier, so that they will be fetched together. Not sure how we can get to that, first, I'll remove the `package:intl` dependency, so that we don't need to worry about that...